index.html

```diff
@@ -1288,6 +1286,137 @@ <h2>DID Subject</h2>
+        true. A malicious <a>DID controller</a> may attempt to impersonate a
+        <a>DID subject</a> by making a false <code>sameSubject</code> assertion.
+        Therefore it is STRONGLY RECOMMENDED that a requesting party obtain
+        independent verification of a <code>sameSubject</code> assertion. One
+        method for doing this is to resolve the <code>sameSubject</code> <a>DID</a>
+        to its own <a>DID document</a> and then verify that it contains the
+        inverse <code>sameSubject</code> assertion (i.e., a backpointer). See
+        the <a href="#security-considerations"></a> and
+        <a href="#privacy-considerations"></a> sections for more information.
+        </p>
+        <p class="note">
+        A <code>sameSubject</code> assertion is not the same as an
+        <code>owl:sameAs</code> assertion. In other words, it does not assert
+        that two or more RDF graphs represented as <a>DID documents</a> can be
+        merged. It only asserts that two or more <a>DIDs</a> identify the same
+        <a>DID subject</a>.
+        </p>
+        </section>
+
+      <section>
+        <h2>representation</h2>
+        <p>
+       As described in Appendix A, a <a>DID</a> is an abstract identifier that
+       does not resolve directly to a <a>representation</a> of the resource
+       identified by the <a>DID</a> (the <a>DID subject</a>). Rather the
+       <a>DID resolution</a> process produces a <a>DID document</a> which in
+       turn describes the <a>DID subject</a>.
+        </p>
+        <p>
+        Appendix A classifies the <a>resources</a> identified by <a>DIDs</a>
+        into three broad categories. In the first two, the <a>DID subject</a>
+        subject is not an information resource but an entity that exists
+        independent of a digital <a>representation</a> on a network. In the
+        third, the <a>DID subject</a>  is an information resource that has a
+        digital <a>representation</a>. Common examples of this type of resource
+        include web pages, PDF files, data schemas, or digital files of any
+        type.
+        </p>
+        <p>
+        In this third case, a <a>DID controller</a> MAY use the
+        <code>representation</code> property to: 1) include a <a>representation</a>
+        of the <a>DID subject</a> directly within the <a>DID document</a>,
+        and/or 2) reference a <a>representation</a> of the <a>DID subject</a>
+        directly from the <a>DID document</a>. Since an information resource
+        may have multiple <a>representations</a>, these two options are not
+        mutually exclusive.
+        </p>
+
+        <dl>
+        <dt>representation</dt>
+        <dd>
+        The <code>representation</code> property MAY have three optional
+        subproperties: <code>type</code>, <code>uri</code>, and <code>value</code>.
+        </dd>
+        </dl>
+
+        <dl>
+        <dt>type</dt>
```

Comment:
`type` is being re-used in a way that is not compliant with the DID spec... this really should be `mediaType` and the value isn't a URI.

Reply:
I agree with that, but also want to mention that this PR seems to serve a similar purpose as various web linking and discovery technologies such as JRD and the HTML <link> element. Both of those use `type` to declare a media type, not an RDF class.

Reply:
if we have an opportunity to pick a better name, we should... `id` and `type` are used all over the place... +1 for not continuing to overload them.... `Content-Type` isn't `type`... https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type

Reply:
Waiting on this to be resolved to do a re-review of this PR.